games-util/lutris/lutris-0.4.23.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1 gnome2-utils xdg-utils
+
+DESCRIPTION="An open source gaming platform for GNU/Linux"
+HOMEPAGE="https://lutris.net/"
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/lutris/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/lutris/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+DEPEND="
```

Comment:
This being a python package, I doubt these are needed in the build process which is probably why *0.4.14* does not have a `DEPEND` at all. E.g. my system installed this package fine without *xgamma* and *python-evdev* present. So, unless there are signs to the contrary please drop `DEPEND` and make them all `RDEPEND` instead, as in the existing ebuild.